React/Modules/RCTDevSettings.mm

```diff
@@ -16,6 +16,8 @@
 #import "RCTProfile.h"
 #import "RCTUtils.h"
 
+#import <React/RCTDevMenu.h>
```

Comment:
Is this okay? This header is from the `DevSupport` target. Otherwise I could check if `bridge.devMenu` exists at runtime.